src/test/java/LoginTests.java

```diff
@@ -18,4 +20,30 @@ public static void LoginEmptyEmailPasswordTest () {
         Assert.assertEquals(driver.getCurrentUrl(), url);
         driver.quit();
     }
-}
+
+    @Test
+    public static void  RegisterToKoel() {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/registration.php";
+        driver.get(url);
+
+        WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
+        emailField.click();
+        emailField.sendKeys("demo99@class.com");
+
+        WebElement loginField = driver.findElement(By.cssSelector("[type='submit'"));
+        loginField.click();
+
+        // WebElement message = driver.findElement(By.className("h3"));
+        // Assert.assertTrue(message.isDisplayed());
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

![image](https://user-images.githubusercontent.com/104761482/209774010-c470c318-24a5-43dc-9d65-f57b9b858ab3.png)